SitumWayfinding/Classes/WayfindingLibrary/LibrarySettings.swift

```diff
@@ -32,6 +32,9 @@ import GoogleMaps
     /// Boolean to configure if location system should use the remote configuration (true) or not (false). See https://situm.com/docs/07-remote-configuration/ to learn more on how to use this functionality.
     private (set) var useRemoteConfig: Bool = false
     
+    /// Boolen configure if show name POI above it
+    private (set) var showTextPois: Bool? = true
```

Comment:
Boolean that configure if the name of the POIs is shown above its icons when painting them on the map